manifest.yml.j2

```diff
@@ -128,6 +128,9 @@ applications:
       FLASK_APP: application.py
       NOTIFY_ENVIRONMENT: {{ environment }}
 
+      NOTIFY_RUNTIME_PLATFORM: paas
+      NOTIFY_REQUEST_LOG_LEVEL: INFO
```

Comment:
These are being set as environment variables here in the manifest, but I don't think they get set on the `app.config` object. I don't see anywhere in the API repo doing that, or in the utils directory.

Therefore, my guess is that this won't take affect as you intend?

Reply:
Yup, changed.